test/functionalTest/cases/0981_POST_v2_attributes/append_and_append_only_combinations.test

```diff
@@ -262,15 +262,15 @@ Date: REGEX(.*)
 
 09. POST /v2/entities/E1/attrs?options=append attribute 'attr4+attr5' (attr4 already exists. error)
 ===================================================================================================
-HTTP/1.1 400 Bad Request
-Content-Length: 108
+HTTP/1.1 422 Unprocessable Entity
```

Comment:
This change in .test (and some other similar) corresponds to the following entry in CNR: "Fix: NGSIv2 "attribute already exists" error case was wrongly using 400 Bad Request, now it uses 422 Unprocessable Entity (according to rules defined in #1286)".

NTC (just informative)